openedx_webhooks/info.py

```diff
@@ -111,4 +118,11 @@ def _is_pull_request(pull_request, kind):
         # This person's institution has the flag.
         return True
 
+    if person.get("before"):
+        for before_date in sorted(person.get("before").keys()):
+          if created_at.date() <= before_date:
+            if person.get("before")[before_date].get("institution") in the_orgs:
+              # This person was an employee of edX when this PR was created, but has since left
+              return True
+
```

Comment:
We talked yesterday about moving this logic.  It would be good to make a new function that takes a person and a date, and returns the person data structure as of that date.  That is a good unit to be a function all its own, and then it can be separately tested.